dist/api_data.py

```diff
@@ -1857,6 +1857,9 @@ def __ge__(self, other):
 ]),
 
 'WT_SESSION.checkpoint' : Method([
+    Config('checkpoint_cleanup', 'false', r'''
```

Comment:
Similar to the RTS PR with multithreads, why don't we enable this by default?

Reply:
Furthermore, could it be a number so we have N threads doing it? I'd rather anticipate the future than have an API change later down the road. We could set the max to 1 for now.

Reply:
Checkpoint cleanup is an operation that involves traversing all the btree to find out the obsolete pages, and sometimes it involves reading internal pages. We do not find any need for multiple workers to perform the checkpoint cleanup.

Also, the above option is for the checkpoint command to trigger the checkpoint cleanup, this option is used only for testing purposes.

Reply:
Thank you, understood.
As discussed, I think it would fit better in a `debug` subsection so we can add more debug options in the future and it is clear that it is for debugging purposes.